### PR TITLE
Include vegans

### PR DESCRIPTION
**Pull request by joshuaflanagan**

### CODE REVIEW

File: dairy.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ yello
 cheese
 goat
 yogurt
+soymilk
```
Comment (joshuaflanagan) [inline dairy.txt:8]: That is weird. are you sure that belongs in dairy?

Comment (joshuaflanagan) [inline dairy.txt:8]: Should we include coconut too?

**Review by joshuaflanagan — COMMENTED** [review]
comments included inline

File: dairy.txt (hunk 1 above)
Comment (joshuaflanagan) [reply dairy.txt:8]: Probably not

**Review by flimflan — APPROVED** [review]
I think its ~good enough~ shippable

File: dairy.txt (hunk 1 above)
Comment (joshuaflanagan) [inline dairy.txt:8]: This is a comment outside of a review

Comment (flimflan) [inline dairy.txt:8]: This is a comment as part of a review

**Review by flimflan — APPROVED** [review]
This is the approval comment

**Pull request merged**